PhysicsTools/PatAlgos/plugins/PatMuonMerger.cc

```diff
@@ -0,0 +1,116 @@
+#include "FWCore/Framework/interface/stream/EDProducer.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/Utilities/interface/InputTag.h"
+
+#include "DataFormats/PatCandidates/interface/Muon.h"
+#include "DataFormats/MuonReco/interface/Muon.h"
+#include "DataFormats/PatCandidates/interface/PackedCandidate.h"
+
+#include "CommonTools/Utils/interface/StringCutObjectSelector.h"
+#include "DataFormats/Common/interface/getRef.h"
+#include "DataFormats/Common/interface/RefToPtr.h"
+
+namespace pat {
+  typedef edm::Ptr<pat::PackedCandidate> PackedCandidatePtr;
+}
+
+class PatMuonMerger : public edm::stream::EDProducer<> {
```

Comment:
All other classes here have "PAT" with all capital letters in their name: please stick on that implicit rule